src/components/Recipes/recipeCard/index.jsx

```diff
@@ -1,12 +1,18 @@
 import React, { useState } from "react";
-import { FullRecipe, RecipeIcons } from "../../../components";
-import {  useNavigate } from "react-router-dom";
+import { FullRecipe, RatingStars, RecipeIcons } from "../../../components";
+import { useNavigate } from "react-router-dom";
+import { ROUTES } from "../../../constants";
 
-export const RecipeCard = ({ recipe, mode = "all-recipes", page, category }) => {
-  const [showFullRecipe, setShowFullRecipe] = useState(false)
-  const navigate = useNavigate()
+export const RecipeCard = ({
+  recipe,
+  mode = "all-recipes",
+  page,
+  category,
+}) => {
+  const [showFullRecipe, setShowFullRecipe] = useState(false);
+  const navigate = useNavigate();
```

Comment:
It`s not clear by its name that this state represents a boolean value.

You can change it into something like `isFullRecipeVisible` for example, indicating to the reader that this is a boolean value.